<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>td_createclass</class>
 <widget class="QWidget" name="td_createclass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>745</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 253, 252);
color: rgb(0, 0, 0);</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>600</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit, QComboBox {
    border: 2px solid #bdc3c7;  
    border-radius: 5px;         
    padding: 5px;              
    background-color: #ffffff;   
    selection-background-color: #3498db;
}


QLineEdit:focus, QComboBox:focus {
    border: 2px solid #3498db;  
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QWidget" name="verticalWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>400</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border color : black;
</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="styleSheet">
            <string notr="true">font: 18pt &quot;Segoe UI&quot;;
color: rgb(206, 187, 255);
color: rgb(0, 0, 0);</string>
           </property>
           <property name="text">
            <string>Create a tuition class</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);
border:none;
border-radius:50px;</string>
           </property>
           <property name="text">
            <string>Tution Title:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="createclass_tuitioname">
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);</string>
           </property>
           <property name="placeholderText">
            <string>Enter your tuition title</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_12">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);
border:none;
border-radius:50px;</string>
           </property>
           <property name="text">
            <string>Tution ID:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="createclass_tuitionid">
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);</string>
           </property>
           <property name="placeholderText">
            <string>Create tuition id</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_13">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);</string>
           </property>
           <property name="text">
            <string>Department</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="createclass_department">
           <property name="font">
            <font>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);</string>
           </property>
           <property name="placeholderText">
            <string>Select your department</string>
           </property>
           <item>
            <property name="text">
             <string>Science</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Commerce</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_10">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);</string>
           </property>
           <property name="text">
            <string>Class</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="createclass_class">
           <property name="font">
            <font>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);</string>
           </property>
           <property name="placeholderText">
            <string>Select your class</string>
           </property>
           <item>
            <property name="text">
             <string>11</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>12</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_9">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);
border:none;
border-radius:50px;</string>
           </property>
           <property name="text">
            <string>Section</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="createclass_section">
           <property name="font">
            <font>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);</string>
           </property>
           <property name="placeholderText">
            <string>Select your section</string>
           </property>
           <item>
            <property name="text">
             <string>A</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>B</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_11">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);</string>
           </property>
           <property name="text">
            <string>Subject</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="createclass_subject">
           <property name="font">
            <font>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);</string>
           </property>
           <property name="placeholderText">
            <string>Select your subject</string>
           </property>
           <item>
            <property name="text">
             <string>Physics</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Math</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Chemistry</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Biology</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Account</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Economics</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Finance</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Business</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);</string>
           </property>
           <property name="text">
            <string>Description</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="createclass_description">
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);</string>
           </property>
           <property name="placeholderText">
            <string>Enter your tuition description</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);</string>
           </property>
           <property name="text">
            <string>Date</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="dateEdit">
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);</string>
           </property>
           <property name="text">
            <string>Time</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTimeEdit" name="timeEdit">
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="createclassbutton">
           <property name="font">
            <font>
             <pointsize>15</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">
background-color: rgb(134, 160, 255);
color: rgb(0, 0, 0);</string>
           </property>
           <property name="text">
            <string>Create Class</string>
           </property>
           <property name="autoDefault">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cancel_button">
           <property name="styleSheet">
            <string notr="true">
background-color: rgb(255, 110, 74);
color: rgb(0, 0, 0);</string>
           </property>
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
